packages/web/src/scss/components/Pagination/README.md

```diff
@@ -0,0 +1,67 @@
+# Pagination
+
+First item selected:
+
+```html
+<div class="Pagination">
+  <a href="#" class="Button Button--secondary Button--square Button--disabled">
+    <svg class="Icon" width="24" height="24" aria-hidden="true">
```

Comment:
Class Icon doesn't exist anymore. Please remove it.

Reply:
Removed.